core/org/commcare/core/parse/ParseUtils.java

```diff
@@ -0,0 +1,35 @@
+package org.commcare.core.parse;
+
+import org.commcare.core.interfaces.AbstractUserSandbox;
+import org.commcare.data.xml.DataModelPullParser;
+import org.javarosa.xml.util.InvalidStructureException;
+import org.javarosa.xml.util.UnfullfilledRequirementsException;
+import org.xmlpull.v1.XmlPullParserException;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ * Created by wpride1 on 8/11/15.
+ */
+public class ParseUtils {
+
+    public static void parseIntoSandbox(InputStream stream, AbstractUserSandbox sandbox) throws InvalidStructureException {
+        parseIntoSandbox(stream, sandbox, false);
+    }
+
+    public static void parseIntoSandbox(InputStream stream, AbstractUserSandbox sandbox, boolean failfast)
+            throws InvalidStructureException {
+        CommCareTransactionParserFactory factory = new CommCareTransactionParserFactory(sandbox);
+        try {
+            DataModelPullParser parser = new DataModelPullParser(stream, factory, failfast, true);
+            parser.parse();
+        } catch (IOException e){
+            e.printStackTrace();
```

Comment:
All of these should be in the signature of the method, not swallowed and ignored.